dev/WindowsAppRuntime.ProxyStub/WindowsAppRuntime.idl

```diff
@@ -16,13 +17,32 @@ interface IDynamicDependencyDataStore : IUnknown
     HRESULT GetPackageFullName([out, retval] LPWSTR* packageFullName);
 };
 
+[object]
```

Comment:
Feels a little strange to _move_ this file here, rather than `#include`-ing it? Or having the common proxystub implementation reference the `_p.c` files and compile them?

Reply:
Yes. I don't understand why this was _moved_ (ditto the coclass below). You're implying we should only have 1 .idl for the whole project. Most unwise.